Earthfile

```diff
@@ -1,44 +1,50 @@
-VERSION --pass-args --use-function-keyword 0.7
-ARG gradle_version=8.5.0
-FROM gradle:${gradle_version}-jdk17
-RUN apt-get update && apt-get install -y \
-  zip \
-  && rm -rf /var/lib/apt/lists/*
+VERSION --pass-args --global-cache 0.7
+IMPORT github.com/earthly/lib/gradle:dbf7cbb9115c02809ebd7cea217bee955ddde6d8 AS gradle
+ARG --global gradle_version=8.2.1
 ARG --global bundle="github.com/earthly/earthfile-grammar+export/"
```

Comment:
Moved it back here since it looks more readable to me having the constants up there